build.cake

```diff
@@ -58,28 +58,29 @@ Task("Clean")
     .Description("Cleans directories.")
     .Does(() =>
     {
-        CleanDirectory(PROJECT_DIR + "bin/");
-        CleanDirectory(PROJECT_DIR + "packages/");
+        CleanDirectory(BIN_DIR);
+        CleanDirectory(PACKAGE_DIR);
         CleanDirectory(IMAGE_DIR);
         CleanDirectory(EXTENSIONS_DIR);
         CleanDirectory(PACKAGE_DIR);
     });
 
-Task("DeleteObjectDirectories")
-    .WithCriteria(BuildSystem.IsLocalBuild)
+Task("CleanAll")
+    .Description("Cleans both Debug and Release Directories followed by deleting object directories")
     .Does(() =>
     {
-        Information("Deleting object directories");
+        Information("Cleaning both Debug and Release");
+        CleanDirectory(PROJECT_DIR + "bin");
```

Comment:
So "Clean" only cleans the current configuration, but "CleanAll" cleans both?

Reply:
Exactly.